doc/user/install.rst

```diff
@@ -94,17 +94,3 @@ Installing khmer inside the virtualenv
    reactivate the virtualenv first::
 
       source khmerEnv/bin/activate
-
-Run the tests
-^^^^^^^^^^^^^
-
-After installing you can run the embedded test suite. If you are running an
-OSX system you should also add `!linux` to prevent linux-specific tests from
-failing.::
-
-      nosetests khmer --attr '!known_failing,!huge'
-
-If the nosetests binary isn't installed then::
-
-      pip2 install khmer[tests]
-      nosetests khmer --attr '!known_failing,!huge'
```

Comment:
This just worked for me.

```bash
deactivate
virtualenv ../temp-env
source ../temp-env/bin/activate
make install-dep
make clean
make install
cd ..
py.test --pyargs khmer.tests -m 'not known_failing and not jenkins and not huge and not linux'
```

I should clarify: by "this worked" I mean "the tests ran". Nine sandbox tests failed, as well as the perennial "C version == Py version" test. But that could've been me.